src/main/java/net/coderbot/iris/samplers/IrisSamplers.java

```diff
@@ -115,6 +115,7 @@ public static boolean addShadowSamplers(SamplerHolder samplers, ShadowMapRendere
 	public static void addLevelSamplers(SamplerHolder samplers, AbstractTexture normals, AbstractTexture specular) {
 		samplers.addExternalSampler(TextureUnit.TERRAIN.getSamplerId(), "tex", "texture", "gtexture");
 		samplers.addExternalSampler(TextureUnit.LIGHTMAP.getSamplerId(), "lightmap");
+		samplers.addExternalSampler(TextureUnit.OVERLAY.getSamplerId(), "overlay");
```

Comment:
This should probably be `iris_overlay` or something, because the `overlay` sampler isn't part of the shaders standard, and is only available on MC 1.15+. In addition, we should reject shaders in `AttributeShaderTransformer` if they contain `iris_` before patching.